### PR TITLE
Issue #3 - Make a POST request to webcompat.com with issue details

### PR DESCRIPTION
**Pull request by ksy36**
r? @denschub

### CODE REVIEW

File: module_shim.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+var module = {};
```
Comment (ksy36) [inline module_shim.js:3]: Borrowed this from webcompat-addon to make it  so `module` do not error out in the browser (added `module.exports` so I could test some functions)

File: moveToWebcompat.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
+"use strict";
+
+const WEBCOMPAT_ENDPOINT = "https://webcompat.com/issues/new";
+const BUGZILLA_ORIGIN = window.location.origin;
+const FALLBACK_MESSAGE = `More information is available on ${window.location}`;
+
 const REQUIRED = {
   op_sys: "OS (Categories > Platform > OS)",
-  platform: "Platform (Categories > Platform)",
   url: "URL (References > URL)",
```
Comment (ksy36) [inline moveToWebcompat.js:9]: I have decided to not make Platform a required field since Browser and OS are sufficient enough

File: utils.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+/**
+ * Returns a string containing version in NN.N format
+ *
+ * @param n
+ * @param os
+ * @returns {string}
+ */
+
+function getVersionNum(n, os) {
+  if (os.includes("iOS")) return iOSFixedFirefoxVersion;
```
Comment (ksy36) [inline utils.js:53]: I've hardcoded this as I'm not sure where to get recent Firefox iOS version. This api only has version of OS https://product-details.mozilla.org/1.0/mobile_versions.json

Comment (skyschub) [reply utils.js:53]: Could you file an issue to investigate that eventually? :)

File: test/utils.test.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+const utils = require("../utils");
+
+global.fetch = jest.fn(() =>
+    Promise.resolve({
```
Comment (skyschub) [inline test/utils.test.js]: Are the four-spaces indentation here expected? I think the usual codestyle is 2-spaces, but prettier might not run on `.test.js` files.

**Review by skyschub — APPROVED** [review]
Assuming this works (which I can't really test right now for funny reasons), this is looking good. Some nits.

File: utils.js (hunk 3 above)
Comment (ksy36) [reply utils.js:53]: Yeah, filed https://github.com/webcompat/bugzilla-to-webcompat/issues/8

File: test/utils.test.js (hunk 4 above)
Comment (ksy36) [reply test/utils.test.js]: Oh right, looks like prettier didn't format this file

**Comment by ksy36** [comment]
Thanks!

**Pull request merged**